src/frontend/komponenter/Fagsak/Vilkårsvurdering/VilkårsvurderingSkjemaInstitusjon.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { Alert } from '@navikt/ds-react';
+import { NavdsSpacing14, NavdsSpacing8 } from '@navikt/ds-tokens/dist/tokens';
+import { RessursStatus } from '@navikt/familie-typer';
+
+import { useVilkårsvurdering } from '../../../context/Vilkårsvurdering/VilkårsvurderingContext';
+import { PersonType } from '../../../typer/person';
+import type { IPersonResultat } from '../../../typer/vilkår';
+import { annenVurderingConfig, AnnenVurderingType, vilkårConfig } from '../../../typer/vilkår';
+import PersonInformasjon from '../../Felleskomponenter/PersonInformasjon/PersonInformasjon';
+import SamhandlerInformasjon from '../../Felleskomponenter/SamhandlerInformasjon/SamhandlerInformasjon';
+import { useSamhandlerRequest } from '../InstitusjonOgVerge/useSamhandler';
+import GeneriskAnnenVurdering from './GeneriskAnnenVurdering/GeneriskAnnenVurdering';
+import GeneriskVilkår from './GeneriskVilkår/GeneriskVilkår';
+import Registeropplysninger from './Registeropplysninger/Registeropplysninger';
+
+const vilkårTilVurdering = [
```

Comment:
Tenker litt på om denne også burde bo i vilkårconfig. Hva tenker du om det?

Reply:
Ja, enig 👍

Reply:
Endte opp med å bruke vilkårconfig på samme måte som før, for så å bytte ut det ene vilkåret med endret ordlyd bare.
Like greit det, kanskje...

Reply:
Hmm, jeg likte det bedre slik det var, egentlig (gitt at listen ligger i `vilkår.ts`). Er ikke helt trygg på at den funksjonen som bytter ut det ene vilkårets ordlyd er like forvaltbar (mtp lesbarhet og kodeforståelse for fremtidige utviklere) som en eksplisitt liste. Men hører gjerne hva @idaame tenker her